[18.0][MIG] sale_team_payment_term: Migration to 18.0 (#4216)
OCA/sale-workflow: sale_team_payment_term/readme/DESCRIPTION.md

> This module allows to set payment term by sales team, if no one exist on

Reviewer: ```suggestion
This module allows to set payment term by sales team, if none is set on
```

Author: ✅ Done !

Thank you @rousseldenis
